modules/twinkleblock.js

```diff
@@ -181,6 +204,32 @@ Twinkle.block.callback.change_action = function twinkleblockCallbackChangeAction
 			tooltip: 'You can use relative times, like "1 minute" or "19 days", or absolute timestamps, "yyyymmddhhmm" (e.g. "200602011405" is Feb 1, 2006, at 14:05 UTC).',
 			value: Twinkle.block.field_block_options.expiry || Twinkle.block.field_template_options.template_expiry
 		});
+
+		if (partialBox) { // Partial block
+			field_block_options.append({
+				type: 'select',
+				multiple: true,
+				name: 'pagerestrictions',
```

Comment:
If anything, the others here are snake, but no, because of how `blockOptions` is handled, the names here serve as the exact parameters passed to the api.  It's quite a bit different, but rather smart IMO.

Reply:
Ah, I see.